RELEASE_NOTES.md

```diff
@@ -7,13 +7,21 @@
 ## Upgrading
 
 * Remove `_soc` formula from the LogicalMeter. This feature has been moved to the BatteryPool.
+* BatteryStatus to track that component stopped sending messages. If battery or its adjacent inverter stopped sending messages, then battery would be considered as not working. (#207)
+* PowerDistributing to send battery status to subscribed users (#205)
 
 ## New Features
 
 * A new class `SerializableRingbuffer` is now available, extending the `OrderedRingBuffer` class with the ability to load & dump the data to disk.
 * Add the `run(*actors)` function for running and synchronizing the execution of actors. This new function simplifies the way actors are managed on the client side, allowing for a cleaner and more streamlined approach. Users/apps can now run actors simply by calling run(actor1, actor2, actor3...) without the need to manually call join() and deal with linting errors.
 * The datasourcing actor now automatically closes all sending channels when the input channel closes.
+* BatteryPool implementation for aggregating battery-inverter metrics into higher level metrics. (#205)
```

Comment:
btw I think `(#205)` won't be a link when the markdown file is rendered. Is this expected? Otherwise you'll need to write it like `[#205](https://github.com/frequenz-floss/frequenz-sdk-python/pull/205)`

Reply:
It does get rendered in the GitHub releases.